pandas/core/arraylike.py

```diff
@@ -274,8 +274,13 @@ def reconstruct(result):
         result = getattr(ufunc, method)(*inputs, **kwargs)
     else:
         # ufunc(dataframe)
-        mgr = inputs[0]._mgr
-        result = mgr.apply(getattr(ufunc, method))
+        if method == "__call__":
```

Comment:
can you make this an elif / else, nesting elif is never the answer

Reply:
I would personally prefer to keep the nesting, as now this full block handles the "ufunc(dataframe)" case (unary ufuncs with dataframe input), and otherwise that case is spread over multiple blocks. I find it clearer to follow this way.